NAMESPACE

```diff
@@ -29,8 +29,15 @@ export(module_aglu_L2252.land_input_5_irr_mgmt_DISABLED)
 export(module_aglu_LB151.ag_MIRCA_ctry_C_GLU_irr)
 export(module_emissions_L111.nonghg_en_R_S_T_Y_DISABLED)
 export(module_energy_L222.en_transformation_DISABLED)
+<<<<<<< HEAD
+export(module_energy_L224.heat)
+export(module_energy_L225.hydrogen_DISABLED)
+export(module_energy_L226.en_distribution_DISABLED)
+export(module_energy_L252.transportation_DISABLED)
+=======
 export(module_energy_L224.heat_DISABLED)
 export(module_energy_L254.transportation_UCD_DISABLED)
+>>>>>>> master
```

Comment:
@jhoring Did you see this git conflict marker? There's no way this is building without an error. Please, when you merge in `master`, look for conflicts and fix them! I'm happy to go over how on Monday.